tests/integration/synapseclient/models/synchronous/test_agent.py

```diff
@@ -0,0 +1,190 @@
+"""Integration tests for the synchronous methods of the AgentSession and Agent classes."""
+
+import pytest
+
+from synapseclient import Synapse
+from synapseclient.models.agent import Agent, AgentSession, AgentSessionAccessLevel
+
+# These are the ID values for a "Hello World" agent registered on Synapse.
+# The Bedrock agent is hosted on Sage Bionetworks AWS infrastructure.
+# CFN Template:
+# https://raw.githubusercontent.com/Sage-Bionetworks-Workflows/dpe-agents/refs/heads/main/client_integration_test/template.json
+CLOUD_AGENT_ID = "QOTV3KQM1X"
+AGENT_REGISTRATION_ID = "29"
+
+
+class TestAgentSession:
+    """Integration tests for the synchronous methods of the AgentSession class."""
+
+    @pytest.fixture(autouse=True, scope="function")
+    def init(self, syn: Synapse) -> None:
+        self.syn = syn
+
+    def test_start(self) -> None:
```

Comment:
Can you make all of the integration tests `async def ...`

The reason for this is described here: https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/CONTRIBUTING.md#lessons-learned-for-integration-testing